dune

```diff
@@ -0,0 +1,8 @@
+(env
+ (dev
+  (flags (:standard -w +42 -ccopt -L/usr/local/lib))
+  (c_flags (:standard -O2 -Wall -Wextra -g -L/usr/local/lib))
+  (env-vars (LD_LIBRARY_PATH /usr/local/lib)))
+ (release
+  (flags (:standard -warn-error))
+  (c_flags (:standard -O2 -Werror))))
```

Comment:
If that was just me, I would remove most of this...

Reply:
What do you don't like here :)?

Reply:
I'd just keep:

```dune
(env
 (dev
  (flags (:standard -w +42))
  (c_flags (:standard -Wall -Wextra -g)))
```

And ` -Wall -Wextra` could also go directly in other dune files.